prov/shm/src/smr_comp.c

```diff
@@ -96,47 +98,51 @@ int smr_rx_comp(struct smr_ep *ep, void *context, uint64_t flags, size_t len,
 		slist_insert_tail(&entry->list_entry,
 				  &ep->util_ep.rx_cq->oflow_err_list);
 		comp->flags = UTIL_FLAG_ERROR;
-	} else {
+	} else if (flags & FI_REMOTE_CQ_DATA || msg_flags & FI_COMPLETION) {
```

Comment:
This suggests that flags and msg_flags shared a common set of bits, which would allow them to be OR'ed